airlock/file_browser_api.py

```diff
@@ -21,6 +21,10 @@ class PathItem:
     container: AirlockContainer
     relpath: pathlib.Path
 
+    # The currently selected path in this container is current the selected.
+    # Used to calculate if how relpath relates to it.
+    selected: pathlib.Path = None
```

Comment:
Should/can selected be an attribute of the container rather than this PathItem? It doesn't feel like a PathItem should be holding information about the selected path in its container.

Reply:
I did wonder that, but what path is currently selected is not something the Workspace or ReleaseRequest cares about - its temporary UI state, its never persisted in any way.

And PathItem is designed to be that UI focussed object, so I do think it is in the right place.
I